src/Symfony/Bundle/SecurityBundle/composer.json

```diff
@@ -19,7 +19,8 @@
         "php": ">=5.3.9",
         "symfony/security": "~2.8|~3.0.0",
         "symfony/security-acl": "~2.7|~3.0.0",
-        "symfony/http-kernel": "~2.2|~3.0.0"
+        "symfony/http-kernel": "~2.2|~3.0.0",
+        "symfony/polyfill-php70": "~1.0"
```

Comment:
It is called in [UserPasswordEncoderCommand.php#L167](https://github.com/symfony/symfony/blob/2.8/src/Symfony/Bundle/SecurityBundle/Command/UserPasswordEncoderCommand.php#L167)